hack/update_releases.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Usage example:
+# export RELEASES_SELECTOR="{0.89.3,0.91.1,0.93.0,0.95.0,99.0.0}"
+# ./hack/update_releases.sh
+
+set -ue
+RELEASES=${RELEASES_SELECTOR//[\{\}]}
+
+IFS=',' read -ra RELEASES_ARRAY <<< "$RELEASES"
+
+for ((i = 0; i < ${#RELEASES_ARRAY[@]} - 1; i++)); do
+    VERSION=${RELEASES_ARRAY[i]}
+    if [ -d "manifests/cluster-network-addons/${VERSION}" ]; then
+      continue
+    fi
+    echo "Processing release: ${VERSION}"
+
+    mkdir -p manifests/cluster-network-addons/${VERSION}
```

Comment:
Please consider Double quote to prevent globbing and word splitting ([SC2086](https://www.shellcheck.net/wiki/SC2086))

```suggestion
    mkdir -p manifests/cluster-network-addons/"${VERSION}"
```

Reply:
There is no risk here in globbing and word splitting as the VERSION is known x.y.z format
Imo it makes the line less readable

Reply:
It's common practice. IMO we should have all our scripts pass shell check, to prevent issues that may occur in the future (even if they don't manifest now).

Reply:
Atm we don't have shell check nor enforcing it, also shell check sometimes even produce bugs btw.
I am against adding it please.
It makes the code less readable, and not mandatory here, as the VERSION is a "fixed", deterministic string.

Reply:
I agree that shell check is not perfect at all. I'm not suggesting to add a shell-check lane to check all scripts, nor do I think we should rework all the existing ones. 
However, I do think we should use it as a tool on new scripts, to make sure we are not falling to known issues, and by that strive to have our scripts as good as possible.

And as for this case - like you said it's not a problem right now so I don't consider it a major issue - and as such I am not going to block over it.